contrib/test/run_test_vectors.sh

```diff
@@ -40,4 +40,7 @@ cat contrib/test/elf-loader-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_
 LOG=$LOG_PATH/test_exec_instr
 cat contrib/test/instr-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_instr --log-path $LOG --log-level-stderr 4
 
+LOG=$LOG_PATH/test_vm_validate
+cat contrib/test/vm_validate-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```

Comment:
```suggestion
<contrib/test/vm_validate-fixtures.list xargs ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```

Reply:
or 
```suggestion
xargs -a contrib/test/vm_validate-fixtures.list ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```
(saves a pipe)